pkgs/development/libraries/miopen/default.nix

```diff
@@ -179,22 +172,14 @@ in stdenv.mkDerivation (finalAttrs: {
 
   postInstall = ''
     rm $out/bin/install_precompiled_kernels.sh
+    mv ../doc/html $out/share/doc/miopen-${if useOpenCL then "opencl" else "hip"}
+    mv ../doc/pdf/miopen.pdf $out/share/doc/miopen-${if useOpenCL then "opencl" else "hip"}
```

Comment:
Should they be moved to `$doc/`?

Reply:
Yes.

Reply:
If you meant "Shouldn't", then they already are in the fixup phase.

Reply:
Ah, so it happens automatically? Makes sense then.